deployment/ansible/group_vars/all.example

```diff
@@ -7,21 +7,24 @@ pfb_aws_batch_job_definition_name: "staging-pfb-analysis-run-job"
 
 # application settings
 docker_compose_version: 1.8.0
-postgresql_database_name: "pfb"
-# User 'gis' allows the Toole analysis scripts to work as is
-postgresql_database_user: "gis"
-postgresql_database_password: "gis"
-postgresql_database_host: "127.0.0.1"
+
+# azavea.docker
+# 1.12.* to match AWS ECS
+docker_version: 1.12.*
+docker_options: "--storage-opt dm.basesize=20G"
+
+# azavea.terraform
+terraform_version: 0.8.7
+
+
+## PostgreSQL and PostGIS parameters ##
```

Comment:
What's the intent for keeping postgresql/postgresql-support/postgis options around? All of the deps are now encapsulated by Docker and we don't install the roles that these variables are used in anymore.